bin/strata-cli/src/constants.rs

```diff
@@ -4,17 +4,22 @@ use alloy::consensus::constants::ETH_TO_WEI;
 use bdk_wallet::bitcoin::{Amount, Network};
 
 /// Number of blocks after bridge in transaction confirmation that the recovery path can be spent.
-pub const RECOVER_DELAY: u32 = 1008;
+///
+/// 144 is the number of blocks in a day.
+pub const RECOVER_DELAY: u32 = 144;
+
+/// Number of block that we consider a transaction "buried" or final.
+pub const FINALITY_DEPTH: u32 = 6;
 
 /// Number of blocks after which we'll actually attempt recovery. This is mostly to account for any
 /// reorgs that may happen at the recovery height.
-pub const RECOVER_AT_DELAY: u32 = RECOVER_DELAY + 10;
+pub const RECOVER_AT_DELAY: u32 = RECOVER_DELAY + FINALITY_DEPTH;
 
 pub const RECOVERY_DESC_CLEANUP_DELAY: u32 = 100;
 
-/// 10 BTC + 0.01 to cover fees in the following transaction where the operator spends it into the
-/// federation.
-pub const BRIDGE_IN_AMOUNT: Amount = Amount::from_sat(1_001_000_000);
+/// 10 BTC + 1,000 satoshi to cover fees in the following transaction where the bridge spends it
+/// into the federation.
+pub const BRIDGE_IN_AMOUNT: Amount = Amount::from_sat(1_000_001_000);
```

Comment:
Is it really not possible/feasible to extract these from the params file?

Reply:
Probably, but we can do it as a refactor for T2?

Reply:
https://alpenlabs.atlassian.net/browse/STR-1156